Sources/DeepLearning/Loss.swift

```diff
@@ -16,14 +16,38 @@
 import TensorFlow
 #endif
 
+/// Computes the Mean Squared Error loss between logits and labels
```

Comment:
Please end each doc comment with a period.
```suggestion
/// Computes the mean squared error between logits and labels.
```

Also, I think you can drop the word "loss".
Example from [TensorFlow Python docs](https://www.tensorflow.org/api_docs/python/tf/nn/softmax_cross_entropy_with_logits_v2):
>  Computes softmax cross entropy between logits and labels.

Thank you!

Reply:
Done